Rakefile

```diff
@@ -22,11 +23,11 @@ def create_trips
     Trip.create(row)
   end
   puts "Imported Trips to Table."
-end
+ end
 
 def create_stations
```

Comment:
Could you update the create_stations method in your branch to be consistent with master. For example :
 ```
SmarterCSV.process('db/csv/station.csv').each do |row|
     Station.create(id: row[0], name: row[1], dock_count: row[4], city: row[5], installation_date: Date.strptime(row[6], "%m/%d/%Y"))
  end
  puts "Imported Stations to Table."
```